### PR TITLE
bugfix api document module update api detail error

### PR DESCRIPTION
**Pull request by tomsun28**
bugfix can not update api detail on api document module.
exception below happen when update api detail.

```
Caused by: org.h2.jdbc.JdbcSQLIntegrityConstraintViolationException: NULL not allowed for column "DATE_CREATED"; SQL statement:
INSERT INTO tag_relation
        (id,date_created,date_updated,api_id,tag_id)VALUES (?,?,?,?,?) [23502-200]
```

Make sure that:

- [x] You have read the [contribution guidelines](https://shenyu.apache.org/community/contributor-guide).
- [x] You submit test cases (unit or integration tests) that back your changes.
- [x] Your local test passed `./mvnw clean install -Dmaven.javadoc.skip=true`.

### CODE REVIEW

File: shenyu-admin/src/main/java/org/apache/shenyu/admin/service/impl/ApiServiceImpl.java [hunk 1]
```diff
@@ -113,6 +113,7 @@ private String update(final ApiDTO apiDTO) {
                     .id(UUIDUtils.getInstance().generateShortUuid())
                     .apiId(apiDO.getId())
                     .tagId(tagId)
+                    .dateCreated(currentTime)
```
Comment (dragon-zhang) [inline shenyu-admin/src/main/java/org/apache/shenyu/admin/service/impl/ApiServiceImpl.java:116]: Please fix the SQL statement, changing the create time in the update interface is incorrect.

Comment (tomsun28) [reply shenyu-admin/src/main/java/org/apache/shenyu/admin/service/impl/ApiServiceImpl.java:116]: ok

Comment (tomsun28) [reply shenyu-admin/src/main/java/org/apache/shenyu/admin/service/impl/ApiServiceImpl.java:116]: > Please fix the SQL statement, changing the create time in the update interface is incorrect.

```
  tagRelationMapper.deleteByApiId(apiDO.getId());
  tagRelationMapper.batchInsert(tags);
```
hi, but due the tag relation always be deleted and then insert new tag relation when update this api, the createdTime always is currentTime.

Comment (dragon-zhang) [reply shenyu-admin/src/main/java/org/apache/shenyu/admin/service/impl/ApiServiceImpl.java:116]: oh, I saw that.

**Review by dragon-zhang — APPROVED** [review]

**Pull request merged**